samples/WebcamServer/component.mk

```diff
@@ -1,5 +1,3 @@
-#ARDUINO_LIBRARIES := ArduCAM
-
-COMPONENT_DEPENDS := WebCam
+ARDUINO_LIBRARIES := WebCam
```

Comment:
@mikee47 what bothers me is the name of the variable `ARDUINO_LIBRARIES`. Maybe we should deprecate it and add an alias called `LIBRARIES`. I am open to suggestions though.

Reply:
I think we should keep ARDUINO_LIBRARIES if only for the sanity of users!

We can't just keep adding stuff to the Libraries (and samples) directory indefinitely, it's feeling a bit unweildy already! I'll open an issue to discuss this.